google/resource_compute_instance_test.go

```diff
@@ -749,7 +813,7 @@ func testAccCheckComputeInstanceDisk(instance *compute.Instance, source string,
 		}
 
 		for _, disk := range instance.Disks {
-			if strings.LastIndex(disk.Source, "/"+source) == len(disk.Source)-len(source)-1 && disk.AutoDelete == delete && disk.Boot == boot {
+			if strings.HasSuffix(disk.Source, source) && disk.AutoDelete == delete && disk.Boot == boot {
```

Comment:
Won't stripping the slash prefixing source mean that a disk with name `bar` would be matched against a disk named `foobar`? Seems a stretch, but...